osquery/tables/system/darwin/signature.mm

```diff
@@ -23,6 +22,42 @@
 namespace osquery {
 namespace tables {
 
+// Get the flags to pass to SecStaticCodeCheckValidityWithErrors, depending on
+// the OS version.
+Status getVerifyFlags(SecCSFlags& flags) {
+  using boost::lexical_cast;
+  using boost::bad_lexical_cast;
+
+  static SecCSFlags sFlags;
+
+  if (sFlags == 0) {
+    auto qd = SQL::selectAllFrom("os_version");
+    if (qd.size() != 1) {
+      return Status(-1, "Couldn't determine OS X version");
+    }
+
+    int minorVersion;
+    try {
+      minorVersion = lexical_cast<int>(qd.front().at("minor"));
+    } catch (const bad_lexical_cast& e) {
+      return Status(-1, "Couldn't determine OS X version");
+    }
+
+    // TODO(andrew-d): This should be tested on a 10.8 machine in order to
```

Comment:
Thanks! One last nit!

You can remove these comment/todo lines, thankfully, we don't support 10.8 (and don't plan on ever supporting it). :P